Binary files ct-capacity-daemon.orig/lib/CtCapacityDaemonNumberCruncher.so and ct-capacity-daemon/lib/CtCapacityDaemonNumberCruncher.so differ
diff -ur ct-capacity-daemon.orig/lib/cruncher.rb ct-capacity-daemon/lib/cruncher.rb
--- ct-capacity-daemon.orig/lib/cruncher.rb	2014-11-11 09:47:50.000000000 +0000
+++ ct-capacity-daemon/lib/cruncher.rb	2022-11-03 15:35:09.413031171 +0000
@@ -12,7 +12,7 @@
         loop do
           start = Time.now
           DaemonKit::logger.info("Cruncher starting")
-          eui = Phoenix::IntfUtils::eui('eth1').upcase
+          eui = Phoenix::IntfUtils::eui('eth1').upcase rescue 'UNKNOWN'
 
           begin
             # generate rack results
diff -ur ct-capacity-daemon.orig/lib/socket_listener.rb ct-capacity-daemon/lib/socket_listener.rb
--- ct-capacity-daemon.orig/lib/socket_listener.rb	2014-09-26 14:14:19.000000000 +0000
+++ ct-capacity-daemon/lib/socket_listener.rb	2022-11-03 16:53:00.630575748 +0000
@@ -8,7 +8,7 @@
     include Phoenix::Logger::Helpers(DaemonKit, 'CtCapacityDaemon::SocketListener')
   
     def start(port, data_store)
-      i "Starting socket listen server.."
+      i "Starting socket listen server on port #{port}..."
       server = TCPServer.new(port)
       Thread.new do 
         begin
